src/java.base/share/man/java.md

```diff
@@ -4004,6 +4004,110 @@ JVM will execute without loading any CDS archives. In addition, if
 you try to create a CDS archive with any of these 3 options specified,
 the JVM will report an error.
 
+## Ahead-of-Time Cache
+
+The JDK supports ahead-of-time (AOT) optimizations that can be performed before an
+application is executed. One example is Class Data Sharing (CDS), as described above,
+that parses classes ahead of time. AOT optimizations can improve the start-up and
+warm-up performance of Java applications.
+
+The Ahead-of-Time Cache (AOTCache) is a container introduced in JDK 24 for
+storing artifacts produced by AOT optimizations. The AOTCache currently contains
+Java classes and heap objects. The plans is to include other types of artifacts,
+such as execution profiles and compiled methods, in future JDK releases.
+
+An AOTCache is specific to a combination of the following:
+
+-   A particular application (as expressed by `-classpath`, `-jar`, or `--module-path`.)
+-   A particular JDK release.
+-   A particular OS and CPU architecture.
```

Comment:
What about VM's flags/settings?

Reply:
The impact of flags is duscissed in the "loading of an AOTCache can fail for a number of reasons" section below. Also, I chose not to enumerate all the possible flag incompatibilities, as there's a pretty exhaustive list in [JEP 483](https://openjdk.org/jeps/483).